src/features/projectsV2/ProjectDetails/components/PlantLocationInfoSection.tsx

```diff
@@ -0,0 +1,94 @@
+import { useMemo } from 'react';
+import * as turf from '@turf/turf';
+import {
+  PlantLocation,
+  SamplePlantLocation,
+} from '../../../common/types/plantLocation';
+import styles from '../styles/PlantLocationInfo.module.scss';
+import ImageCarousel from './ImageCarousel';
+import TreePlantedData from './microComponents/TreeCountData';
+import SpeciesPlanted from './microComponents/SpeciesPlanted';
+import SampleSpecies from './microComponents/SampleSpecies';
+import TreeMapperBrand from './microComponents/TreeMapperBrand';
+import PlantingDetails from './microComponents/PlantingDetails';
+import { useTranslations } from 'next-intl';
+
+const PlantLocationInfoSection = ({
+  plantLocationInfo,
+}: {
+  plantLocationInfo: PlantLocation | SamplePlantLocation | null;
+}) => {
+  const isMultiTreeRegistration =
+    plantLocationInfo?.type === 'multi-tree-registration';
+  const tProjectDetails = useTranslations('ProjectDetails');
+  const { totalTreesCount, plantedLocationArea } = useMemo(() => {
+    const totalTreesCount = isMultiTreeRegistration
+      ? plantLocationInfo.plantedSpecies.reduce(
+          (sum, species) => sum + species.treeCount,
+          0
+        )
+      : 0;
+    const area = plantLocationInfo?.geometry
+      ? turf.area(plantLocationInfo?.geometry)
+      : 0;
+    const plantedLocationArea = area / 10000;
+    return { totalTreesCount, plantedLocationArea };
+  }, [plantLocationInfo?.geometry, plantLocationInfo?.type]);
+
+  const plantingDensity = totalTreesCount / plantedLocationArea;
+
+  const sampleInterventionSpeciesImages = useMemo(() => {
+    if (isMultiTreeRegistration) {
+      const result = plantLocationInfo.sampleInterventions.map((item) => {
+        return {
+          image: item.coordinates[0].image,
+          description: tProjectDetails('sampleTreeTag', { tag: item.tag }),
+        };
+      });
+      return result;
+    }
+  }, [isMultiTreeRegistration ? plantLocationInfo.sampleInterventions : null]);
+
+  const hasSampleInterventionSpeciesImages =
+    sampleInterventionSpeciesImages !== undefined &&
+    sampleInterventionSpeciesImages?.length > 0;
+
+  return (
+    <section className={styles.plantLocationInfoSection}>
+      <TreePlantedData
+        plHid={plantLocationInfo?.hid}
+        totalTreesCount={totalTreesCount}
+        plantedLocationArea={plantedLocationArea}
+      />
+      {hasSampleInterventionSpeciesImages && (
+        <ImageCarousel
+          images={sampleInterventionSpeciesImages}
+          type={'coordinate'}
+          imageSize={'large'}
+          imageHeight={195}
+          leftAlignment={15}
+        />
```

Comment:
The text here needs to have some overlay behind - currently it is often hard to distinguish above the image.

Reply:
I have fixed this  in the sample plant location  (projects-redesign/sample-plant-location) PR